src/simulator_core/entities/assets/pipe.py

```diff
@@ -84,6 +84,8 @@ def _get_diameter(self, esdl_asset: EsdlAssetObject) -> float:
         """Retrieve the diameter of the pipe and convert it if necessary."""
         temp_diameter, property_available = esdl_asset.get_property("innerDiameter", self.diameter)
         if property_available:
+            if temp_diameter == 0:
```

Comment:
Misschien is het beter om hier PIPE_DEFAULTS.diameter terug te geven?

Reply:
Aangepast